tasks/modules.py

```diff
@@ -278,6 +278,7 @@ def dependency_path(self, agent_version):
         targets=["./pkg/runner", "./pkg/utils/e2e/client"],
         lint_targets=[".", "./examples"],  # need to explicitly list "examples", otherwise it is skipped
     ),
+    "test/otel": GoModule("test/otel", independent=True, used_by_otel=True),
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution. -->
**❓ question**
Is this used by `otel`?

Reply:
Technically not, we can remove `used_by_otel` if you prefer. I just added it to include it in the otel module version check.

Reply:
Thank you, then I would rename `used_by_otel` to `include_in_otel_module_check`, or something similar.

Reply:
`used_by_otel` is also used by ASC in [update_go_mods](https://github.com/DataDog/datadog-agent/blob/5998a28ed6a4f0f96ef7152ff0093c8b53c0ef64/tasks/update_go.py#L191), so I'm inclined to keep this name. We can change it in a separate PR if preferred though